packages/api/src/libs/utils.ts

```diff
@@ -205,6 +205,13 @@ export function getWeb3(url: string, options: Obj = {}) {
   throw new Error("Web3 provider URLs not supported");
 }
 
+export function getEthers(url: string) {
+  if (url.startsWith("ws")) return new ethers.providers.WebSocketProvider(url);
+  if (url.startsWith("http")) return new ethers.providers.JsonRpcProvider(url);
```

Comment:
Might also want to check https

Reply:
It's covered by `startsWith("http")` I think, right?